docs/features/index.rst

```diff
@@ -1,8 +1,12 @@
 Features
 ========================
 
+This page lists the major ScyllaDB features that help you achieve high
+performance and optimize workloads.
```

Comment:
On second thought, this wording is not helping.
Some of these features (e.g., MV) are unrelated to performance or workload optimization.

Reply:
Agree. Lets keep the second paragraph only.